1-org/modules/network/variables.tf

```diff
@@ -64,3 +64,9 @@ variable "project_budget" {
   })
   default = {}
 }
+
+variable "auto_create_network" {
+  description = "Control whether the 'default' network will be created in the project."
+  type        = bool
+  default     = true
+}
```

Comment:
this should not be a variable, it is a hard coded value when using the project factory method